src/styles/shelley/tabs.st.css

```diff
@@ -0,0 +1,60 @@
+/* tabs.st.css */
+/* = Shelley project scope and vars */
+@st-import Shelley, [
+    --color-ui-inverse,
+    --color-ui-raw,
+    --color-accent-1
+  ] from "./project.st.css";
+/* Component API */
+@st-import Tabs from "../../components/Tabs/tabs.st.css";
+
+/* = This is how we can scope the styles in Stylable. */
```

Comment:
1. Are these comments necessary?

2. In this 1 PR in 2 `st.css` files we have a different style of commenting. For instance here we say "Shelley" and have an extra comment for scoping (which we should know since we work with Stylable).

Reply:
<img width="489" alt="Screenshot 2023-05-17 at 12 22 03 PM" src="https://github.com/action-is-hope/shelley/assets/16628652/32545ab0-4ae2-49ec-957e-0e2f7c7414fd">

Reply:
Yep, you are right to bring up consistency and the usefulness of comments in general. It would be good to come together as a group and define this as part of the contrib stuff and maybe a ref component to follow which has examples of comment usage. #93 

In general these type of repetitive comments can go and be part of the docs. I guess I was imagining the theme code to be self documenting in the sense that someone could come in without much knowledge of stylable and see what we are doing but it's a bit annoying after a while. Some generic text at the top of theme files which point to some docs on how to get started with stylable perhaps would be nice.